wallet/src/Pos/Wallet/Web/State/Transactions.hs

```diff
@@ -74,9 +85,9 @@ applyModifierToWallet2 walId wAddrs custAddrs utxoMod
                       currentBlockchainDepth syncState = do
     case syncState of
         (WS.RestoringFrom rhh newSyncTip) -> do
-            for_ wAddrs WS.addWAddress
+            for_ wAddrs WS.addWAddress2
```

Comment:
The remaining changes in this module will not be needed once we don't have `addWAddress2` etc.